docs/versioning_releases.rst

```diff
@@ -0,0 +1,52 @@
+Versioning and releases
```

Comment:
```suggestion
Versioning and releaseing
```
Ditto the file name, wdyt?

Reply:
Thanks, but I think "releases" work better here.